app/controllers/shelters_controller.rb

```diff
@@ -18,4 +18,15 @@ def create
               zip: params[:shelter_zip])
     redirect_to '/shelters'
   end
+
+  def edit
+  end
+
+  def update
+    shelter = Shelter.find(params[:id])
+    shelter.update({name: params[:shelter_name],
+      address: params[:shelter_address], city: params[:shelter_city],
+      state: params[:shelter_state], zip: params[:shelter_zip]})
+    redirect_to "/shelters/#{shelter.id}"
```

Comment:
- find the right shelter to update based on passed in parameter
- .update passes in the given parameters and saves it to the database
- redirect to the show page